.rubocop.yml

```diff
@@ -6,3 +6,9 @@ AllCops:
     - 'config/**/*'
     - 'spec/spec_helper.rb'
     - 'vendor/**/*'
+
+Metrics/LineLength:
+  Max: 120
+
+Style/StringLiterals:
+  EnforcedStyle: single_quotes
```

Comment:
@jchristo4 this only applies to ruby style enforcement.  We need to add the following to `.scss-style.yml` to affect the enforcement of scss rules.

      StringQuotes:
        style: single_quotes